Sources/SymbolKit/SymbolGraph/Symbol/KindIdentifier.swift

```diff
@@ -23,6 +23,8 @@ extension SymbolGraph.Symbol {
         case `func`
         case `operator`
         case `init`
+        case ivar
```

Comment:
Should we consider converting these two statically defined properties to avoid API breakage like this in the future?

It doesn't seem like we should be promising a totally stable API here via enum cases.

It will be API breaking either way since clients will need to add a new `default` case but at least we'll avoid breaking API in the future.

Reply:
+1 here. I think we need to add a default unknown kind to avoid these sorts of breakages.

Reply:
I like this suggestion! I imagine we'll want to add more symbol kinds in the future, and implying that consumers can exhaustively match them is perhaps a little misleading, if we want to properly support _any_ symbol kind that a potential symbol graph could offer (cf. #7).

Could we change Swift-DocC to use `case default` branches first, though? It should still work with an enum, and the DocC code defaults to an `.unknown` symbol kind anyway. Then we could land this type definition change to `KindIdentifier` here without breaking DocC or toolchain builds.